controls/pdfviewer/search/styling.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Styling
+page_title: .NET MAUI PDF Viewer Documentation - Styling
+description: Learn more about the styling options for the search toolbar items.
+position: 0
+slug: pdfviewer-search-styling
+---
+
+# .NET MAUI PDF Viewer Search Commands
+
+The PDF Viewer for .NET MAUI provides a flexible styling API for its search toolbar items. 
+
+As the `PDFViewerToolbar` is based on the `RadToolbar` control, all toolbar items in the PDF Viewer inherit from `ButtonToolbarItem`. All styling properties available for the `ButtonToolbarItem` are also applicable for the PDF toolbar items. 
+
+The table below lists all toolbar items related to the search feature and their target type:
+
+| Toolbar Item | Target Type |
+| ------ | ------ |
+| `PdfViewerSearchNavigationToolbarItem` | `Style` property with target type `NavigationButtonToolbarItemView`  |
+| `PdfViewerNavigateToPreviousSearchResultToolbarItem` | `Style` property with target type `PdfViewerNavigateToPreviousSearchResultToolbarItemView` |
+| `PdfViewerNavigateToNextSearchResultToolbarItem` | `Style` property with target type `PdfViewerNavigateToNextSearchResultToolbarItemView` |
+| `PdfViewerSearchEntryToolbarItem` | `Style` property with target type `PdfViewerSearchEntryToolbarItemView` |
+| `PdfViewerSearchBusyIndicatorToolbarItem` | `Style` property with target type `PdfViewerSearchBusyIndicatorToolbarItemView` |
+
+The `PdfViewerSearchBusyIndicatorToolbarItemView` inherits from the [BusyIndicatorToolbarItemView]({%slug toolbar-items-busyindicator%}) article.
```

Comment:
Inherits from an article?

Reply:
i refactored the sentence more than 3 times, so article left my mistake